automations/justfile

```diff
@@ -8,7 +8,7 @@ default:
 # Dev #
 #######
 
-# Install Python dependencies in Pipenv environments
+# Install Python dependencies in Pipenv environments and JS dependencies
```

Comment:
Did we want to move this into the `automations` folder? It seems like it would be better to have this at the top level, but I could understand not wanting to have it there.

Reply:
I'm conflicted on this too. I'll wait out for more feedback!

Reply:
I think the `justfile` should be in the repo root, we can use it for a lot more than just automations (like linting RFCs).